Lib/test/test_dataclasses.py

```diff
@@ -2883,6 +2883,11 @@ class C:
                 # x is not an InitVar, so there will be a member x.
                 self.assertEqual(C(10).x, 10)
 
+    def test_initvar_repr(self):
+        self.assertEqual(repr(InitVar[str]), 'dataclasses.InitVar[str]')
```

Comment:
There is already a test for `repr()` in `test_init_var_preserve_type`. I suggest to remove it from that test.